src/components/PackageFiles.tsx

```diff
@@ -30,6 +34,17 @@ export function PackageFiles({arch, pkgname, repo}: PackageFilesProps) {
     }
   };
 
+  if (isArchlinux) {
+    return (
+      <a
+        className="text-primary hover:underline"
+        href={`https://archlinux.org/packages/${repo}/${arch}/${pkgname}/`}
```

Comment:
```suggestion
        href={`https://archlinux.org/packages/${repo}/${arch}/${pkgname}/files`}
```

that way we can redirect directly to the files listing